packages/data-explorer/src/views/MainView.vue

```diff
@@ -73,7 +73,13 @@ export default Vue.extend({
       get: function () {
         return this.$store.state.tablePagination
       },
-      set: function (value) { this.setPagination(value) }
+      set: function (value) {
+        this.$router.push({
```

Comment:
It seems that there are about 3 router pushes to get to the table's final state. 
It may be cleaner to have a watcher on a common table state object that includes 
all properties (page, size, sort, filter) and throttle route changes on nextTick?

Reply:
i think you want to use the information contained in the event source ( this gets lost when using watchers ). I agree we could propagate all route change event to the top but suggest to lease that for another PR, this PR already solves a lot of router/state issues